GetIntoTeachingApi/Models/GetIntoTeaching/Validators/TeachingEventSearchRequestValidator.cs

```diff
@@ -13,9 +13,9 @@ public TeachingEventSearchRequestValidator(IStore store)
                 .MaximumLength(40)
                 .Matches(Location.OutwardOrFullPostcodeRegex)
                 .Unless(request => request.Postcode == null && request.Radius == null);
-            RuleFor(request => request.TypeId)
-                .SetValidator(new PickListItemIdValidator<TeachingEventSearchRequest>("msevtmgt_event", "dfe_event_type", store))
-                .Unless(request => request.TypeId == null);
+            RuleFor(request => request.TypeIds)
```

Comment:
`FluentValidation` has a `RuleForEach` that will let you use the current validator that works on a single pick list item id, for example:

```
RuleForEach(r => r.TypeIds)
  .SetValidator(new PickListItemIdValidator<TeachingEventSearchRequest>("msevtmgt_event", "dfe_event_type", store));
```